src/fundamental/macro-fundamental.h

```diff
@@ -50,6 +50,15 @@
 #define UNIQ_T(x, uniq) CONCATENATE(__unique_prefix_, CONCATENATE(x, uniq))
 #define UNIQ __COUNTER__
 
+#define ONCE __ONCE(UNIQ_T(_once_, UNIQ))
+#define __ONCE(o)                            \
+        ({                                   \
+                static bool (o) = true;      \
```

Comment:
hmm, this should probably be inverted, because zero-initialized variables can be put in the bss section and thus don't need to be included in the ELF image.

Reply:
ack, will do